### PR TITLE
Update index.html.

### PR DESCRIPTION
**Pull request by tclaar**
Improve HTML structure, modify spacing and add line breaks where needed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -97,12 +97,18 @@
...
-
-    A and D for left and right.<br>Space for jump and double jump.<br>Shift for Sprint.<br>While mid-air pressing A or D Hold X to Dash.<br>Left Click to shoot.
+    <div id="options">
+        <label for="debug">Debug</label>
+        <input type="checkbox" id="debug" name="debug" />
+        <label for="spanish">Spanish</label>
+        <input type="checkbox" id="spanish" name="spanish" />
+    </div>
+    
+    <p>A and D for left and right.</p>
```
Comment (b1gd3vd0g) [inline index.html:107]: Thanks for putting in the paragraph tags!

**Review by b1gd3vd0g — APPROVED** [review]
This is very good stuff! No notes

File: index.html [hunk 2]
```diff
@@ -97,12 +97,18 @@
...
+    <p>Space for jump and double jump.</p>
```
Comment (calebkrauter) [inline index.html:108]: Looks good!

**Review by calebkrauter — APPROVED** [review]
Looks good!

**Pull request merged**